deploy/kubernetes/rbac.yaml

```diff
@@ -42,6 +42,9 @@ rules:
   - apiGroups: [""]
     resources: ["events"]
     verbs: ["list", "watch", "create", "update", "patch"]
+  - apiGroups: ["storage.k8s.io"]
```

Comment:
Add a comment that this is only required if enabling the alpha volume modify feature

Reply:
Can you also verify that with the feature gate disabled in the resizer, and without this rbac rule, that the resizer is able to come up successfully?

Reply:
Added to the test case description in the PR.

Reply:
Can you also add a comment to this rbac file?

Reply:
Updated all three comments in [this commit](https://github.com/kubernetes-csi/external-resizer/pull/351/commits/83e82b2dadca7aa4a11e33fdb40c4076178f2bd3).